apps/paf/translations/src/en/fields.json

```diff
@@ -1334,7 +1334,7 @@
         "label": "Other"
       },
       "gender-unknown": {
```

Comment:
change this to `prefer-not-to-say` here and in fields/index.js

Reply:
Changed to prefer-not-to-say